src/styles/_catalog.scss

```diff
@@ -1,22 +1,32 @@
+$card-width: 200px;
+
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-columns: repeat(4, $card-width);
+  gap: 48px;
 
   max-width: 944px;
   margin: 0 auto;
```

Comment:
I think that there should not be "max width" here. Since the width is set by the elements. And the grid has its own option to center the content through justify-content. Then you can get rid of the margin.

Reply:
Max-width helps to set the position of a catalog. Without max-width and margin, it will stick to the left side of the screen